app/src/main/java/com/plaid/linksample/LinkSampleApplication.kt

```diff
@@ -6,9 +6,13 @@ package com.plaid.linksample
 
 import android.app.Application
 import com.plaid.link.Plaid
+import com.plaid.linksample.network.LinkSampleApi
+import com.plaid.linksample.network.LinkSampleRetrofit
 
 class LinkSampleApplication : Application() {
 
+  val linkSampleApi = LinkSampleRetrofit.retrofit.create(LinkSampleApi::class.java)
```

Comment:
I'm not sure if it's worthwhile to avoid unnecessary retrofit API creation here.

On one end it's good to avoid creating the API too often (as it's expensive), but on the other hand this doesn't demonstrate a particular good practice on how to do the scoping either. (e.g. casting the Application class to access the instance).

I would be in favor of creating the retrofit API on demand (e.g. when the `openLink` method is called) and potentially add a comment on top, that retrofit instance creation is expensive and that it's better to do that once and keep it around for as long the process/link activity lives.